packages/radio/src/lib/radio.stories.mdx

```diff
@@ -0,0 +1,125 @@
+import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs/blocks';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import pkg from '../../package.json';
+import Radio from './radio';
+
+<Meta title="Radio" component={Radio} />
+
+# Radio <img src={`https://img.shields.io/badge/Version-v${pkg.version}-brightgreenn`} />
+
+### Installation
+
+First install the latest version of the package from `@novatics/radio`.
+
+```shell
+  yarn add -D @novatics/radio
+```
+
+
+### Dependencies
+
+This component extends [Material UI React Radio](https://mui.com/material-ui/react-radio-button). Therefore is needed to install Material UI:
+
+```bash
+yarn add @mui/material
+```
+
+### Basic Usage
+
+<Canvas>
+  <Story name="Basic Usage">
+    {(args) => {
+      return (
+        <div style={{ display: 'flex', flexDirection: 'row' }}>
+          <Radio
```

Comment:
How about we add some text here to showcase better? I know that maybe FormControlLabel is needed for this, but since there are details about text font on Figma for Radio, I believe we can rethink this approach.